### ISSUE TITLE
Missing boolean/bit data type

### ISSUE THREAD

**Issue by ondras** [ondras/wwwsqldesigner#56]
_From [atrop...@gmail.com](https://code.google.com/u/101131384345788038437/) on November 22, 2009 05:58:00_

The type drop down list doesn't contain a data type for boolean (aka bit) 
data types.

_Original issue: http://code.google.com/p/wwwsqldesigner/issues/detail?id=49_

**Comment by ondras** [comment]
_From [ondrej.zara](https://code.google.com/u/ondrej.zara/) on November 22, 2009 03:34:21_

For which DB? Can you please adjust the datatypes.xml and send me the patch?

**Comment by ondras** [comment]
_From [ober.14@osu.edu](https://code.google.com/u/ober.14@osu.edu/) on December 26, 2009 14:37:25_

atrophic: feel free to let me know what database it is, and I can make a patch if you can't.

**Comment by ondras** [comment]
_From [atrop...@gmail.com](https://code.google.com/u/101131384345788038437/) on December 27, 2009 12:01:08_

Part of this bug was me not reading directions.  I had changed the database I was 
designing against after I started the design, which did not work (as designed).  
However, after starting from scratch, MySQL was still missing the BIT data type, 
which was an alias for tinyint(1) prior to version 5.0.3, but is now an official data 
type (albeit with some confusing implementation).

I added it to the Miscellaneous section, where it is found in most other database 
engines' datatypes.xml.  Attached is my attempt at a patch.

kuba, thanks for reminding me about this ;)

**Attachment:** [mysql_add_bit.patch](http://code.google.com/p/wwwsqldesigner/issues/detail?id=49#hc3)
